applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/status.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
+'require view';
+'require rpc';
+'require poll';
+'require dom';
+'require ui';
+
+ 
+ // https://github.com/openwrt/luci/blob/eabf1d020fad5f866ea6742c02fc808d0c43b349/applications/luci-app-keepalived/htdocs/luci-static/resources/view/keepalived/peers.js#L7
+
+
+return view.extend({
+	
+	callHostHints: rpc.declare({
+		object: 'luci-rpc',
+		method: 'getHostHints',
+		expect: { '': {} }
+	}),
+    callGetRemotehosts: rpc.declare({
+		object: 'usteer',
+		method: 'remote_hosts',
+		expect: {  '': {}}
+	}),
+	callGetRemoteinfo: rpc.declare({
+		object: 'usteer',
+		method: 'remote_info',
+		expect: { '': {} }
+	}),
+	callGetLocalinfo: rpc.declare({
+		object: 'usteer',
+		method: 'local_info',
+		expect: { '': {} }
+	}),
+	callGetClients: rpc.declare({
+		object: 'usteer',
+		method: 'get_clients',
+		expect: { '': {} }
+	}),
+
+
+	load: function() {
+		return Promise.all([
+			this.callHostHints(),
+			this.callGetRemotehosts(),
+			this.callGetRemoteinfo(),
+			this.callGetLocalinfo(),
+			this.callGetClients()
+			
+		]);
+	},
+	handleReset: null,
+	handleSaveApply: null,
+	handleSave: null,
+	
+	render: function(data) {
+	
+		var hosts = data[0];
+		var Remotehosts=data[1];
+		var Remoteinfo=data[2];
+		var Localinfo=data[3];
+		var Clients=data[4];
+
+
+
+		var body = E([
+			E('h2', _('Usteer status'))
+		]);
+
+
+/////////////////////////
+		body.appendChild(
+			E('h3', 'Remotehosts')
+		);
+		var remotehost_table = E('table', { 'class': 'table cbi-section-table' }, [
+			E('tr', { 'class': 'tr table-titles' }, [
+				E('th', { 'class': 'th' }, _('IP address')),
+				E('th', { 'class': 'th' }, _('identifier'))
+			])
+		]);
+		
+		var remotehosttableentries =[];
+		for(var IPaddr in Remotehosts) {
+			remotehosttableentries.push([
+							IPaddr, Remotehosts[IPaddr]['id']
+						]);
+				}	
+
+		cbi_update_table(remotehost_table, remotehosttableentries, E('em', _('No data')));
+		body.appendChild(remotehost_table);
+		
+/////////////////////////
+		body.appendChild(
+			E('h3', 'Client list')
+		);
+		var conenctioninfo_table = E('table', { 'class': 'table cbi-section-table' }, [
```

Comment:
Is the misspelling intentional? 
"conenctioninfo" instead of "connectioninfo" ?
That seems to be in all places in the code...

Reply:
typo, will fix